README.md

```diff
@@ -15,6 +15,17 @@ See the [getting started doc](docs/getting_started.md) on support on how to get
 After installation, refer back to the below sections for supported scenarios and how to tweak the kraken config to load them on your cluster
 
 
+### Setting up infrastructure dependencies
+Kraken indexes the metrics specified in the profile into Elasticsearch and visualizes them in Grafana in addition to leveraging Cerberus for understanding the health of the kubernetes/OpenShift cluster under test. More information on the features is documented below. The infrastruture pieces can be easily installed by running:
+
+```
+$ cd kraken
+$ podman-compose up
+```
+
+This will spin up Elasticsearch, Grafana and Cerberus containers on the host on which you are running Kraken.
+
```

Comment:
Ravi and I talked about this offline, but I had issues with the size of my linux ec2 machine. Might be helpful to mention the [doc](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/recognize-expanded-volume-linux.html) we used to get around this issue

Reply:
Fixed it, thanks.